tests/x11/vlc.pm

```diff
@@ -31,11 +31,13 @@ sub run {
     assert_screen "vlc-network-window";
     send_key "backspace";
     type_string autoinst_url . "/data/Big_Buck_Bunny_8_seconds_bird_clip.ogv";
+    assert_screen "url_check";
     assert_and_click "vlc-play_button";
     # The video is actually 23 seconds long so give a bit of headroom for
     # startup
     assert_screen "vlc-done-playing", 90;
     send_key "ctrl-q";
+    send_key "alt-f4";    # try to quit if ctrl-q doesn't work
```

Comment:
Why not to wrap it with send_key_until_needle match then? We could use generic-desktop needle to verify that window got closed. WDYT?

Reply:
Any progress here? I guess we can use generic-desktop needles to assert closed window.